setup.py

```diff
@@ -13,7 +13,7 @@
 HERE = path.dirname(__file__)
 
 with open(path.join(HERE, "requirements.txt")) as requirements_file:
-    requirements = requirements_file.read().split()
+    requirements = requirements_file.read().strip().split("\n")
```

Comment:
why is this necessary now?

Reply:
When you install DDA with `pip install`, it installs the list of dependencies indicated by `install_requires`, which this is used for further down.  The `example` package installs DDA this way.  It was erroring out when the dependency was spelled:

```
django >=2.2, <4
```

because this parsing didn't handle those spaces correctly.  It's not strictly necessary now that I changed the dependency to

```
django~=3.2
```

but I still think it's a bug in our setup.py, even if it's not triggered now.